src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,156 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTodos, getUserById } from '../../api/api';
+import { TodosStatus } from '../../store';
+import {
+  setTodosAction,
+  setUserAction,
+  setStatusAction,
+  setFilterAction,
+} from '../../store/actions';
+import {
+  getFilterSelector,
+  getStatusSelector,
+  getTodosSelector,
+  getUserSelector,
+} from '../../store/selector';
+
+import './TodoList.scss';
+
+export const TodoList = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(getStatusSelector);
+  const todos = useSelector(getTodosSelector);
+  const user = useSelector(getUserSelector);
+  const filter = useSelector(getFilterSelector);
+
+  useEffect(() => {
+    const loadTodosFromServer = async () => {
+      let todosFromServer = await getTodos();
+
+      todosFromServer = todosFromServer.filter(el => el.title.includes(filter));
```

Comment:
It is better to bring it to one register to search in Lower & UpperCase

Reply:
agree, you should handle lower and upper cases here